backends/firrtl/firrtl.cc

```diff
@@ -599,8 +593,8 @@ struct FirrtlWorker
 					primop = "eq";
 					always_uint = true;
 					firrtl_width = 1;
-				}
-				else if ((cell->type == "$ne") | (cell->type == "$nex")) {
+			    }
```

Comment:
Please use tabs for indentation.

Reply:
Same as above. Fixed now after reading the yosys style guide. Thanks for pointing it out.